lib/queue_classic/tasks.rb

```diff
@@ -11,11 +13,8 @@
     @worker = QC.default_worker_class.new
 
     trap('INT') do
-      $stderr.puts("Received INT. Shutting down.")
-      if !@worker.running
-        $stderr.puts("Worker has stopped running. Exit.")
-        exit(1)
-      end
+      $stderr.puts("Received INT. Shutting down.")      
+      abort("Worker has stopped running. Exit.") unless @worker.running
```

Comment:
We used to also `exit(1)` in this case, but no longer do.

Reply:
Oh wait I missed that the `abort` is new, we were doing the same thing in two steps above. 🙈